openedx/core/djangoapps/schedules/resolvers.py

```diff
@@ -358,6 +359,13 @@ def schedules_for_bin(self):
                 )
                 # continue to the next schedule, don't yield an email for this one
             else:
+                unsubscribe_url = None
+                if 'bulk_email_optout' in settings.ACE_ENABLED_POLICIES:
+                    unsubscribe_url = reverse('bulk_email_opt_out', kwargs={
```

Comment:
`unsubscribe_url` needs to be an absolute URL for the current site.

See [common.djangoapps.student.forms](https://github.com/edx/edx-platform/blob/73af914cfc8c82fa7bd0fc8e5fd0c1eb9b81494c/common/djangoapps/student/forms.py#L70-L82) for example, though this email already has an absolute link in the "Resume your course now", so it may be better to reuse the logic which generates that link instead.

Reply:
Have resolved this by using the `with_link_tracking` templatetag in the template, which makes the URL absolute as well as adding link tracking.